src/scenes/game.py

```diff
@@ -12,10 +14,14 @@
 
 
 class Game(Scene):
-    def __init__(self, screen, scenes, config=None):
+    # pylint: disable=too-many-arguments
```

Comment:
to support the extra `standalone` argument, the `too-many-arguments` error would appear (6/5)